mopidy/audio/actor.py

```diff
@@ -742,25 +754,25 @@ def set_metadata(self, track):
         :param track: the current track
         :type track: :class:`mopidy.models.Track`
         """
-        taglist = gst.TagList()
+        taglist = Gst.TagList.new_empty()
         artists = [a for a in (track.artists or []) if a.name]
 
         # Default to blank data to trick shoutcast into clearing any previous
```

Comment:
Was just playing with this and doesn't seem to work, i.e. we might as well default to nothing